.travis.yml

```diff
@@ -1,7 +1,16 @@
 ---
-sudo: false
 language: ruby
 cache: bundler
 rvm:
-  - 2.4.5
-before_install: gem install bundler -v 1.17.1
+- 2.4.5
+- 2.5.3
+before_install:
+- 'echo ''gem: --no-ri --no-rdoc --no-document'' > ~/.gemrc'
```

Comment:
Any reason this couldn't just be:

```yaml
- 'echo "gem: --no-ri --no-rdoc --no-document" > ~/.gemrc'
```

The double single quotes kinda weird me out (personal problem most likely...)

Reply:
yeah, that's how YAML.dump dumps it, so I'm gonna leave it